### PR TITLE
Enable leading wildcard by default

### PR DESCRIPTION
**Pull request by tarzanek**

### CODE REVIEW

File: opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/Configuration.java [hunk 1]
```diff
@@ -494,6 +494,7 @@ public Configuration() {
         setUserPageSuffix("");
         setWebappLAF("default");
         // webappCtags is default(boolean)
+        setAllowLeadingWildcard(true);
```
Comment (idodeclare) [inline opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/Configuration.java]: Hi. See above @tulinkry’s comment at line 436.

**Comment by tarzanek** [comment]
thanks @idodeclare , fixed the order ;-)

**Comment by vladak** [comment]
indexer in Docker uses the Indexer option. Didn't you want to remove that as well ?

**Comment by vladak** [comment]
So, what is the plan with `--leadingWildCards` going forward ?

**Comment by tarzanek** [comment]
we can leave it for some time as backwards compatibility(since I assume lots of folks do use it in their scripts) and then remove it?

**Comment by vladak** [comment]
file a issue to track it then.

**Comment by idodeclare** [comment]
Maybe let’s leave in perpetuity for folks who want to prevent the execution of costly leading-wildcard queries?

**Comment by vladak** [comment]
Can be. Would be nice to turn the command line option into toggle then.

**Pull request merged**